integ/package.json

```diff
@@ -55,55 +55,55 @@
     "pkglint": "pkglint -f"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "1.102.0",
-    "@types/jest": "^26.0.22",
-    "@types/node": "^14.14.41",
-    "@typescript-eslint/eslint-plugin": "^4.22.0",
-    "@typescript-eslint/parser": "^4.22.0",
-    "aws-cdk": "1.102.0",
-    "eslint": "^7.24.0",
+    "@aws-cdk/assert": "1.104.0",
+    "@types/jest": "^26.0.23",
+    "@types/node": "^15.3.0",
+    "@typescript-eslint/eslint-plugin": "^4.23.0",
+    "@typescript-eslint/parser": "^4.23.0",
+    "aws-cdk": "1.104.0",
+    "eslint": "^7.26.0",
     "eslint-import-resolver-node": "^0.3.4",
     "eslint-import-resolver-typescript": "^2.4.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-license-header": "^0.2.0",
     "jest": "^26.6.3",
     "pkglint": "0.31.0",
-    "ts-jest": "^26.5.4"
+    "ts-jest": "^26.5.6"
   },
   "dependencies": {
-    "@aws-cdk/aws-autoscaling": "1.102.0",
-    "@aws-cdk/aws-docdb": "1.102.0",
-    "@aws-cdk/aws-ec2": "1.102.0",
-    "@aws-cdk/aws-ecr": "1.102.0",
-    "@aws-cdk/aws-ecs": "1.102.0",
-    "@aws-cdk/aws-efs": "1.102.0",
-    "@aws-cdk/aws-elasticloadbalancingv2": "1.102.0",
-    "@aws-cdk/aws-iam": "1.102.0",
-    "@aws-cdk/aws-logs": "1.102.0",
-    "@aws-cdk/aws-route53": "1.102.0",
-    "@aws-cdk/aws-s3": "1.102.0",
-    "@aws-cdk/aws-s3-assets": "1.102.0",
-    "@aws-cdk/aws-secretsmanager": "1.102.0",
-    "@aws-cdk/core": "1.102.0",
+    "@aws-cdk/aws-docdb": "1.104.0",
+    "@aws-cdk/aws-ec2": "1.104.0",
+    "@aws-cdk/aws-ecr": "1.104.0",
+    "@aws-cdk/aws-autoscaling": "1.104.0",
```

Comment:
The ordering was changed here for some reason so it isn't alphabetical anymore.

Reply:
Good catch, not sure why that happened. Might've been changed by `npm-upgrade`.